src/accelerator.rkt

```diff
@@ -134,6 +134,10 @@
 (define-accelerator (fnmadd real real real) real (lambda (x y z) (+ (neg (* x y)) z)))
 (define-accelerator (fnmsub real real real) real (lambda (x y z) (- (neg (* x y)) z)))
 
+(define-accelerator (fmsub real real real) real (lambda (x y z) (- (* x y) z)))
+(define-accelerator (fnmadd real real real) real (lambda (x y z) (+ (neg (* x y)) z)))
+(define-accelerator (fnmsub real real real) real (lambda (x y z) (- (neg (* x y)) z)))
```

Comment:
This looks like duplicates

Reply:
Why? x86 provides all four versions of FMA (as in, with the minus signs in all four possible places) and in principle we might want to leverage that in Herbie (pushing a negation in and thus saving one operation)